base/system/services/database.c

```diff
@@ -1418,23 +1523,24 @@ ScmControlService(HANDLE hControlPipe,
                        &Overlapped);
     if (bResult == FALSE)
     {
-        DPRINT("ReadFile() returned FALSE\n");
+        DPRINT("(%S, %d) ReadFile() returned FALSE\n", lpService->lpServiceName, dwControl);
 
         dwError = GetLastError();
         if (dwError == ERROR_IO_PENDING)
         {
-            DPRINT("dwError: ERROR_IO_PENDING\n");
+            DPRINT("(%S, %d) dwError: ERROR_IO_PENDING\n", lpService->lpServiceName, dwControl);
 
             dwError = WaitForSingleObject(hControlPipe,
                                           PipeTimeout);
-            DPRINT("WaitForSingleObject() returned %lu\n", dwError);
+            DPRINT("(%S, %d) WaitForSingleObject() returned %lu\n", lpService->lpServiceName, dwControl, dwError);
 
             if (dwError == WAIT_TIMEOUT)
             {
+               DPRINT1("(%S, %d) WaitForSingleObject() timed out!\n", lpService->lpServiceName, dwControl, dwError);
```

Comment:
missing space in the beginning (wrong indent). And Superfluous `!` in the end
```suggestion
                DPRINT1("(%S, %d) WaitForSingleObject() timed out\n", lpService->lpServiceName, dwControl, dwError);
```

Reply:
Changed. Thanks.